octoprint_mrbeam/migration/migration_base.py

```diff
@@ -73,13 +75,19 @@ def id(self):
     def shouldrun(cls, beamos_version):
         """
         Checks if this Miration should run
+
         Args:
             cls: Migrationclass
             beamos_version: current beamos_version
 
         Returns:
             bool: True if this migration should run
         """
+        if not isinstance(beamos_version, basestring):
```

Comment:
Very nice selection of the class to compare to .. a link to confirm your good choice  
https://stackoverflow.com/a/1979107